project_report/tex/introduction.tex

```diff
@@ -1 +1,21 @@
 \section{Introduction}
+% Motivation from project proposal
+As students, this semester project gives the group great learning experience as
+well as a good idea of how such a project would progress. During the project,
+the group will learn how to create and set up an Open Platform Communications
+Unified Architecture, OPC-UA, client, and how to make it communicate directly
+to a physical machine. The group will learn to host a web server, from which the
+machine can be controlled, and access it as a website. The group will also have
+the chance to learn and use a new scripting language, JavaScript. The gained
+experience is the main motivation of this project.
+
+% Problem formulering
+
```

Comment:
Maybe add something like this here: https://pastebin.com/i7e01sV7

Reply:
:+1: